docs/reference/components/sinks/console.cue

```diff
@@ -79,4 +79,9 @@ components: sinks: console: {
 			summary:      true
 		}
 	}
+
+	telemetry: metrics: {
+		vector_processed_bytes_total:   telemetry.metrics._vector_processed_bytes_total
```

Comment:
```suggestion
		vector_processed_bytes_total:   _vector_processed_bytes_total
```

Reply:
This unfortunately won't work. `_vector_processed_bytes_total` is essentially "namespaced" under `telemetry.metrics`. Without that, Cue doesn't know "where" they are. I was also hoping that this would work out, but alas.